hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -405,8 +423,15 @@ def button_add_line(self):
         for rec in self:
             if rec.state in ['new', 'draft']:
                 rec.add_line(rec.add_line_project_id, rec.add_line_task_id)
-                rec.add_line_task_id = False
-                rec.add_line_project_id = False
+                rec.reset_add_line()
+        return True
```

Comment:
That's from @alexey-pelykh commit

Reply:
But the module V12 doesn't return True and I'm reviewing this PR that contains a return True. Could you please remove the return and make it consistent with V12?

Reply:
Yeah, I will remove it. In v12 it's not there because you deleted it in https://github.com/OCA/hr-timesheet/commit/6dfe2ef56b48d1d8a8e2b5d7c587f331405a2ecf#diff-a48ce9b933da452cb1910c0e256932c7L392.